app/models/group_member.rb

```diff
@@ -6,6 +6,8 @@ class GroupMember < ApplicationRecord
   has_many :assignments, through: :group
 
   after_commit :send_welcome_email, on: :create
+  scope :is_mentor, -> { where(mentor: true) }
+  scope :is_member, -> { where(mentor: false) }
```

Comment:
scope :mentor, -> { where(mentor: true) }
 scope :member, -> { where(mentor: false) }
 No need of if